src/components/Screens/OurApps.js

```diff
@@ -1,57 +1,98 @@
 import React from "react";
+import axios from "axios";
+import reqwest from "reqwest";
+import Carousel from "nuka-carousel";
 import "src/styles/Screens/OurApps.css";
 
+const settings = {
+  wrapAround: true,
+  speed: 500,
+  slidesToShow: 1
+};
+
 class OurApps extends React.Component {
-  render () {
+  constructor(props) {
+    super(props);
+    this.state = {extract: []};
+  }
+  componentWillMount() {
+    reqwest({
```

Comment:
please refactor this out to separate method and call it within `componentWillMount`